src/main/config/config_streamer.c

```diff
@@ -131,6 +140,16 @@ static uint32_t getFLASHSectorForEEPROM(void)
         return FLASH_SECTOR_6;
     if ((uint32_t)&__config_start <= 0x080FFFFF)
         return FLASH_SECTOR_7;
+#if defined(STM32F765xx)
```

Comment:
(BTW: this can be implemented as array of page boundaries)

Reply:
Sure, I leave it for a PR where scatter loading is done uniformly for all platforms, with as little assembly as possible (or none at all).